core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/convert.kt

```diff
@@ -284,13 +285,38 @@ public fun <T : Any> DataColumn<T?>.convertToBoolean(): DataColumn<Boolean?> = c
 
 // region convert URL
 
-public fun <T, R : URL?> Convert<T, R>.toIFrame(
+@JvmName("toIframeFromUrlNullable")
+@Refine
+@Converter<IFRAME>(IFRAME::class, nullable = true)
```

Comment:
The compiler can infer the `<>` part if you pass a `KClass` right? :)

Reply:
Surprisingly, no. If i remove it, compiler complains about wrong number of type arguments

Reply:
but do you use the type argument? Or could it work if you accept `KClass<*>`?

Reply:
Type argument is not needed. Initially i wanted to only use type argument, but compiler doesn't see it - only KClass get class expression. Get class works as expected without type argument